ubuntusetup/setup.sh

```diff
@@ -10,11 +10,11 @@ ubuntusetup() {
     if [ -e .paperbenni/setup/programs.txt ]; then
         echo "program list found"
     else
-        echo "no list found"
+        echo "no list found" >&2
         return 1
     fi
 
-    for THISPROGRAM in $(cat .paperbenni/setup/programs.txt); do
+    for THISPROGRAM in $(< .paperbenni/setup/programs.txt); do
```

Comment:
Hope that's okay, Meowster. Savings launching yet another process.

Reply:
Cool, didn't know this was a thing.